subscription_management/lambda_function.py

```diff
@@ -101,14 +108,14 @@ def lambda_handler(event, context):
         # A future improvement could be made to authenticate this, but for the moment,
         # I don't forsee anyone DoS-ing this endpoint causing any harm.
         sms_file_id, phone_file_id = upload_telephony_contact_list(body.get('category'))
-        return respond_success("{\"status\": \"success\", \"sms_file_id\": \"%s\", \"phone_file_id\": \"%s\"}" % (sms_file_id, phone_file_id), origin)
+        return respond_success(f"{{\"status\": \"success\", \"sms_file_id\": \"{sms_file_id}\", \"phone_file_id\": \"{phone_file_id}\"}}", origin)
```

Comment:
These lines (both before and after your changes) are really busy. Mind using your branch and PR as an opportunity to split this these into two ops, where one is construction of the json and the second is the call to `respond_success`?

Reply:
Done